examples/object_detection/main.py

```diff
@@ -181,9 +181,14 @@ def criterion_fn(model_outputs, target, criterion):
         with torch.no_grad():
             print_statistics(compression_ctrl.statistics())
             net.eval()
-            mAp = test_net(net, config.device, test_data_loader, distributed=config.distributed)
-            if config.metrics_dump is not None:
-                write_metrics(mAp, config.metrics_dump)
+            if config['ssd_params'].get('loss_inference', False):
+                model_loss = test_net(net, config.device, test_data_loader, distributed=config.distributed,
+                                      loss_inference=True, criterion=criterion)
+                logger.info("Final model loss: {:.3f}".format(model_loss))
+            else:
+                mAp = test_net(net, config.device, test_data_loader, distributed=config.distributed)
+                if config.metrics_dump is not None:
+                    write_metrics(mAp, config.metrics_dump)
```

Comment:
I think that we won't fall into this branch anymore with the config you've changed. We need that config to execute this code branch so that the example script, as run from within the CI validation run, produces an output file with metrics which is then parsed for a test report.

If you modified the config file to give an example of the option's usage, I suggest that you just create another config file in addition to the original one.

Reply:
```config['ssd_params'].get('loss_inference', False)```. When the key is not provided, the app would go as default to mean AP evaluation block. so, the original config and CI validation would still works. does this make sense? or am I disconnected at some points?

Reply:
The config modification is not meant to be as sample, just laying out the key for reference, ie setting it to false so it is backward compatible. Skipping the key is also handled in the main app.